palisade/example-model/src/main/java/uk/gov/gchq/palisade/example/runner/RestExample.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2018 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.palisade.example.runner;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import uk.gov.gchq.palisade.User;
+import uk.gov.gchq.palisade.client.ClientUtil;
+import uk.gov.gchq.palisade.example.client.ExampleSimpleClient;
+import uk.gov.gchq.palisade.example.common.ExampleUsers;
+import uk.gov.gchq.palisade.example.common.Purpose;
+import uk.gov.gchq.palisade.example.hrdatagenerator.types.Employee;
+import uk.gov.gchq.palisade.service.PalisadeService;
+
+import java.util.stream.Stream;
+
+
+public class RestExample {
+    private static final Logger LOGGER = LoggerFactory.getLogger(RestExample.class);
+
+    public static void main(final String... args) throws Exception {
```

Comment:
if this is the class we are running to make each query then this main method needs to be taking in the resource id, user name and the query purpose.

Reply:
This class is a wrapper around ExampleSimpleClient which already contains these variables.